app/timetables/models.py

```diff
@@ -37,8 +37,10 @@ class Meal(SlugifyMixin, models.Model):
     slugify_field = 'name'
 
     def clean(self):
-        if self.start_time >= self.end_time:
-            raise ValidationError(_('start_time must be less than end_time.'))
+        # Ensure start time and end time are not None before compare
+        if self.start_time and self.end_time:
+            if self.start_time >= self.end_time:
```

Comment:
@andela-kerinoso I don't think breaking it into multiple lines defeats the intended purpose. The purpose is to make it more expressive not shorter. It's my 2 cents. But going to what you said `nested if shouldn't be what's stopping this PR.`